installers/msi-language/ActiveState/Tracker.cs

```diff
@@ -110,10 +113,14 @@ public void TrackEventSynchronously(Session session, string msiLogFileName, stri
             session.Log("Sending event {0}/{1}/{2} for cid={3} (custom dimension 1: {4}, pid={5})", category, action, label, this._cid, langVersion, pid);
             var sessionID = computeSessionID(msiLogFileName);
             var t = Task.WhenAll(
-                TrackEventAsync(sessionID, category, action, label, langVersion, value),
+                TrackEventAsync(session, sessionID, category, action, label, langVersion, value),
                 TrackS3Event(session, sessionID, category, action, label)
             );
-            t.Wait();
+            var completed = t.Wait(TimeSpan.FromSeconds(15));
```

Comment:
Isn't this redundant with the client.Timeout?

Reply:
As mentioned during stand-up, the MSI still blocks forever without this "outer" timeout.  Perhaps, because the GA Event blocks forever.  But I couldn't find a way to change the underlying http-client used by that library.

As discussed I suggest to leave the client timeout in there anyways, so we can use it when we add retries in a later story.